src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -160,98 +244,103 @@ export class BucketDetailComponent implements OnInit {
-                  let lastNum = arr.Contents.Key.lastIndexOf(this.colon);
-                  if(lastNum == arr.Contents.Key.length -1){
+
+                  let lastNum = arr.Key.lastIndexOf(this.colon);
+                  if(lastNum == arr.Key.length -1){
+
+
                     folderContain = true;
                   }
                 }
               }
               return folderContain;
             })
             this.allDir.forEach(val=>{
-              val.ObjectKey = val.ObjectKey.slice(this.folderId.length);
+              val.Key = val.Key.slice(this.folderId.length);
             })
           }else{
             //Distinguish between folders and files at the first level
             this.allDir = this.allDir.filter(item=>{
               let folderIndex = false;
-              if(item.Contents.Key.indexOf(this.colon) !=-1){
+
+              if(item.Key.indexOf(this.colon) !=-1){
                 let index;
-                index = item.Contents.Key.indexOf(this.colon,index);
+                index = item.Key.indexOf(this.colon,index);
                 //Distinguish between folders and files in folders
-                if(index == item.Contents.Key.length-1){
+                if(index == item.Key.length-1){
                   folderIndex = true;
                 }
               }
-              return item.Contents.Key.indexOf(this.colon) ==-1 || folderIndex;
+              return item.Key.indexOf(this.colon) ==-1 || folderIndex;
             })
           }
           let folderArray = [];
           this.allFolderNameForCheck = [];
           this.allDir.forEach(item=>{
-            item.size = Utils.getDisplayCapacity(item.Contents.Size,2,'KB');
-            item.lastModified = Utils.formatDate(item.Contents.LastModified *1000);
-            item.Location = item.Contents.Location;
-            item.Tier = "Tier_" + item.Contents.Tier + " (" + item.Contents.StorageClass + ")";
-            if(item.Contents.Key.indexOf(this.colon) !=-1){
-              item.objectName = item.Contents.Key.slice(0,item.Contents.Key.lastIndexOf(this.colon));
+            item.size = Utils.getDisplayCapacity(item.Size,2,'KB');
+            item.lastModified = Utils.formatDate(item.LastModified);
+            item.Tier = item.Key.indexOf(this.colon) == -1 ? "Tier_" + item.Tier + " (" + item.StorageClass + ")" : '--';
+            if(item.Key.indexOf(this.colon) !=-1){
+              item.objectName = item.Key.slice(0,item.Key.lastIndexOf(this.colon));
               this.allFolderNameForCheck.push(item.objectName);
               item.newFolder = true;
               item.disabled = false;
               item.size = "--";
               backupAllDir.forEach(arr=>{
                 if(this.folderId !=""){
-                  let hasFolder = arr.Contents.Key.indexOf(this.folderId) !=-1 && arr.Contents.Key != this.folderId;
+                  let hasFolder = arr.Key.indexOf(this.folderId) !=-1 && arr.Key != this.folderId;
```

Comment:
All the changes related to the change in the response structure have already been merged in #223.
Take the latest development code and rebase.

Reply:
I have restored the previous Key and uploaded it again. Please check it again. Thank you